### PR TITLE
Create `/version` command

### PR DESCRIPTION
**Pull request by ccuser44**

### CODE REVIEW

**Comment by bearbin** [comment]
Looks good :) Could you please add yourself to the CONTRIBUTORS file to indicate that you accept the licensing?

File: cmd_version.lua [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function HandleVersionCommand(Split, Player)
+	-- Send output to player:
+	SendMessage(Player, string.format(
+		"This server is running CuberiteMC %s - %s (%s - %s) with Lua version %s and SQL version %s.",
```
Comment (mathiascode) [inline cmd_version.lua:4]:
```suggestion
		"This server is running Cuberite %s - %s (%s - %s) with Lua version %s and SQL version %s.",
```

File: Info.lua [hunk 2]
```diff
@@ -235,6 +235,23 @@ g_PluginInfo =
...
 
+		["/version"] = 
+		{
+			Alias = "/ver",
+			Permission = "core.version",
+			Handler = HandleVersionCommand,
+			HelpString = "Shows the version of the server.",
+		},
+
+		["/about"] =
```
Comment (mathiascode) [inline Info.lua:246]: Is there a reason why this isn't an alias of /version?

**Review by mathiascode — CHANGES_REQUESTED** [review]

File: Info.lua [hunk 3]
```diff
@@ -235,6 +235,23 @@ g_PluginInfo =
...
+		{
+			Alias = "/ver",
+			Permission = "core.version",
+			Handler = HandleVersionCommand,
+			HelpString = "Shows the version of the server.",
+		},
+
+		["/about"] = 
+		{
+			Alias = "/a",
```
Comment (mathiascode) [inline Info.lua:248]: This seems too generic, unless other server software use it as an alias?

Comment (ccuser44) [reply Info.lua:248]: SpigotMC, PaperSpigit and CraftBukkit use the same alias name.

Comment (ccuser44) [reply Info.lua]: I still decided to change it to /abt

**Comment by ccuser44** [comment]
> Looks good :) Could you please add yourself to the CONTRIBUTORS file to indicate that you accept the licensing?

Done.

File: CONTRIBUTORS [hunk 4]
```diff
@@ -29,4 +29,5 @@ tigerw (Tiger Wang)
 tonibm19
 tonitch (Debucquoy Anthony)
 xoft (Mattes Dolak/madmaxoft)
+ccuser44
```
Comment (madmaxoft) [inline CONTRIBUTORS:32]: Please alpha-sort

File: Info.lua [hunk 5]
```diff
@@ -235,6 +235,23 @@ g_PluginInfo =
...
+			HelpString = "Shows the version of the server software.",
+		},
+
+		["/about"] = 
+		{
+			Alias = "/abt",
+			Permission = "core.version",
+			Handler = HandleVersionCommand,
+			HelpString = "Shows the version of the server software.",
+		},
```
Comment (madmaxoft) [inline Info.lua:252]: Please alpha-sort.

File: Info.lua [hunk 6]
```diff
@@ -926,6 +943,18 @@ g_PluginInfo =
...
+
+		["about"] = 
+		{
+			Handler = HandleVersionCommand,
+			HelpString = "Shows the version of the server software.",
+		},
+
```
Comment (madmaxoft) [inline Info.lua:957]: Please alpha-sort

File: Info.lua [hunk 7]
```diff
@@ -1445,6 +1474,11 @@ g_PluginInfo =
 			Description = "Allows players to manage the whitelist.",
 			RecommendedGroups = "admins",
 		},
+
+		["core.version"] =
+		{
+			Description = "Shows the version of the server software.",
+		},
```
Comment (madmaxoft) [inline Info.lua:1481]: Please alpha-sort

File: README.md [hunk 8]
```diff
@@ -26,6 +26,8 @@ Implements some of the basic commands needed to run a simple server.
 |/motd | core.motd | Shows the message of the day.|
 |/op | core.rank | Add a player to the administrator rank.|
 |/plugins | core.plugins | Shows a list of the plugins.|
+|/version | core.version | Shows the server software version.|
+|/about | core.version | Shows the server software version.|
```
Comment (madmaxoft) [inline README.md:30]: Please alpha-sort.
I believe this is generated, so sorting things in Info.lua should fix this, too.

File: cmd_version.lua [hunk 9]
```diff
@@ -0,0 +1,13 @@
+function HandleVersionCommand(Split, Player)
+	-- Send output to player:
+	SendMessage(Player, string.format(
+		"This server is running Cuberite %s - %s (%s - %s) with Lua version %s and SQL version %s.",
```
Comment (madmaxoft) [inline cmd_version.lua:4]: It's not `SQL` as in the generic sense, it's `SQLite`, a specific library providing SQL-like capabilities on local files. Not sure it warrants reporting its version, considering we don't update too much.

**Review by madmaxoft — CHANGES_REQUESTED** [review]
Please put pretty much everything into an alpha-sorted order. That way things are easier to find later on.

**Review by madmaxoft — APPROVED** [review]
Awesome, thanks.

**Comment by mathiascode** [comment]
CI is failing:
```
Scenario: fuzzing command "/about ".
Calling request "Command /about ".
lua: /home/runner/work/Core/Core/cmd_version.lua:3: bad argument #7 to 'format' (string expected, got function)
stack traceback:
	[C]: in function 'format'
	/home/runner/work/Core/Core/cmd_version.lua:3: in function 'Function'
	Simulator.lua:1391: in function 'processCallbackRequest'
	Simulator.lua:969: in function 'executePlayerCommand'
	Scenario.lua:172: in function 'fuzzSingleCommand'
	Scenario.lua:199: in function 'fuzzCommand'
	Scenario.lua:279: in function 'action'
	Scenario.lua:642: in function 'execute'
	Simulator.lua:1482: in function 'run'
	CuberitePluginChecker.lua:29: in main chunk
	[C]: ?
```

**Review by mathiascode — APPROVED** [review]

**Pull request merged**